packages/default/scss/fab/_theme.scss

```diff
@@ -0,0 +1,54 @@
+@include exports( "fab/theme" ) {
+    // FAB
+    .k-fab {
+        @include box-shadow($fab-shadow);
+    }
+
+    // Theme colors
+    @each $name, $color in $fab-theme-colors {
+        .k-fab-#{$name} {
+            border-color: $color;
+            color: contrast-wcag( $color );
+            background-color: $color;
+            background-image: linear-gradient(rgba( adjust-hue( $color, 14), 0 ), rgba( adjust-hue( $color, 14), .2 ));
+        }
+    }
+
+    // Hover Theme colors
+    @each $name, $color in $fab-theme-colors {
+        .k-state-hover.k-fab-#{$name},
+        .k-fab-#{$name}:hover {
+            border-color: adjust-hue( $color, 3 );
+            color: contrast-wcag( $color );
+            background-color: adjust-hue( $color, 3 );
+            background-image: linear-gradient(rgba( adjust-hue( adjust-hue( $color, 3 ), 14), 0 ), rgba( adjust-hue( adjust-hue( $color, 3 ), 14), .2 ));
+        }
+    }
+
+    // Focus Theme colors
+    @each $name, $color in $fab-theme-colors {
+        .k-state-focus.k-fab-#{$name},
+        .k-fab-#{$name}:focus {
+            box-shadow: 0 3px 4px 0 rgba($color, .4);
```

Comment:
box-shadow can be extracted as a variable

Reply:
I couldn't extract these as variables (I know the buttons has their values extracted) but here we have a map over the themecolors. What will be the `$color` in this case if it's in _variables.scss?